include/api/CAnomalyJob.h

```diff
@@ -224,14 +216,11 @@ class API_EXPORT CAnomalyJob : public CDataProcessor {
     void outputInterimResults(core_t::TTime bucketStartTime);
 
     //! Helper function for outputResults.
-    //! \p processingTimer is the processing time can be written to the bucket
-    //! \p sumPastProcessingTime is the total time previously spent processing
-    //! but resulted in no bucket being outputted.
+    //! \p processingTime is the processing time can be written to the bucket
```

Comment:
is the processing time can be written to the bucket |-> is the processing time of the bucket